src/validators/user/user.validator.js

```diff
@@ -0,0 +1,23 @@
+const { body } = require("express-validator");
+
+const signupValidators = [
+  body("email").isEmail().withMessage("Please provide a valid email"),
+  body("password")
+    .isStrongPassword()
+    .withMessage(
+      "Password must be at least 8 characters long, and contain at least one lowercase letter, one uppercase letter, one number, and one special character."
+    ),
+];
+
+const updatePasswordValidator = [
+  body("password")
+    .optional()
```

Comment:
`password` field optional? Also, shouldn't we take the user's current password, as well as confirm the newer password, from the user before actually updating it based on only a single string?